### ISSUE TITLE
Cargo bay and fighter bay taken/available space is not shown in stats panel

### ISSUE THREAD

**Issue by DarkFenX** [pyfa-org/Pyfa#1877]
Fighters addition pane selected:
![image](https://user-images.githubusercontent.com/275209/54196464-93807100-44d2-11e9-8570-bafafb11d9d0.png)

Cargo addition pane selected:
![image](https://user-images.githubusercontent.com/275209/54196482-9da26f80-44d2-11e9-8e82-29a46b340f7a.png)

**Comment by DarkFenX** [comment]
Looks like it's bug with ResourcesViewFull.toggleContext() code, possibly wx bug in Hide(), Show() and/or ShowItems()

**Comment by DarkFenX** [comment]
@blitzmann does it ever happen on windows?

**Comment by DarkFenX** [comment]
Tracked down what's causing this issue. It's setting label to StaticText while element is hidden. So if you refresh panel while drones are open - drones are refreshed properly, but when you switch to fighters or cargo (hiding drone info and showing fighter/cargo info), this bug occurs.

```
import wx


class MainFrame(wx.Frame):

    def __init__(self):
        wx.Frame.__init__(self, None, wx.ID_ANY, 'test')
        # self.Maximize()

        box = wx.BoxSizer(wx.HORIZONTAL)

        lbl1 = wx.StaticText(self, wx.ID_ANY, "0")
        box.Add(lbl1, 0, wx.ALIGN_CENTER | wx.LEFT, 5)

        box.Add(wx.StaticText(self, wx.ID_ANY, "/"), 0, wx.ALIGN_CENTER)

        lbl2 = wx.StaticText(self, wx.ID_ANY, "0")
        box.Add(lbl2, 0, wx.ALIGN_CENTER)

        self.SetSizer(box)
        self.Show()
        box.ShowItems(False)
        lbl1.SetLabel('1')
        lbl2.SetLabel('1')
        box.ShowItems(True)



app = wx.App(False)
mf = MainFrame()
app.MainLoop()
```
This code reproduces this issue, at least on both linux machines I am using.

**Comment by DarkFenX** [comment]
Filed upstream bug https://github.com/wxWidgets/Phoenix/issues/1182

**Comment by blitzmann** [comment]
@DarkFenX Nope, on windows it works as intended... I'm surprised no one has reported this - it's been ages since we've touched that specific code

**Comment by blitzmann** [comment]
@DarkFenX Ive got a fix in place for this issue, please test. Works for me on my ubuntu VM tho :)

**Comment by DarkFenX** [comment]
Yeah i wanted to try out this fix as well, and maybe resetting label upon showing it in toggleContext(). But if current implementation doesn't flicker, then we could just use it.

**Comment by DarkFenX** [comment]
Changed fix in 5f8ccb98313812ea4e84106c7013e32f736479e5, according to advice from upstream issue.
